src/app/announcements/page.tsx

```diff
@@ -0,0 +1,348 @@
+'use client';
+
```

Comment:
The styles for the calendar inputs are not showing:

![Screenshot 2024-12-17 165607](https://github.com/user-attachments/assets/d7b5e340-55ab-4eff-ab67-2dc621fd0329)

I think there's a css file you have to input from mantine to apply them

Reply:
I think if you add this import: "import '@mantine/dates/styles.css';" it should apply the styles

Reply:
![fixed-calendar](https://github.com/user-attachments/assets/75a6e493-dbba-43dc-8d66-bd780088835a)